css/moonstone-rules.less

```diff
@@ -62,8 +62,6 @@ html {
 }
 
 .matrix3dsurface {
-    -webkit-transform-origin: center center;
```

Comment:
This seems like a weird change to me? Are we sure this will not have impact else where? Maybe a solid commit message for this would be beneficial too.